API/NoDocs/Microsoft.AspNetCore.Telemetry/LoggingOptions.cs

```diff
@@ -0,0 +1,39 @@
+// Assembly 'Microsoft.AspNetCore.Telemetry.Middleware'
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Compliance.Classification;
+using Microsoft.Extensions.Http.Telemetry;
+using Microsoft.Shared.Data.Validation;
+
+namespace Microsoft.AspNetCore.Telemetry;
+
+public class LoggingOptions
+{
+    public bool LogRequestStart { get; set; }
+    public bool LogBody { get; set; }
+    public IncomingPathLoggingMode RequestPathLoggingMode { get; set; }
+    [Experimental("EXTEXP0003", UrlFormat = "https://aka.ms/dotnet-extensions-warnings/{0}")]
```

Comment:
@jeffhandley I thought I posted this but now I can't find it...  hope you don't mind me asking again. do we have a list where one can claim diagnostic ID"s for experimental attributes? seems like we're claiming EXTEXPnnnn ...

I know there's a list for obsoletion diagnostics in the core libraries, but can't find that now.

Reply:
Here ya go, @danmoseley: https://github.com/dotnet/extensions/pull/4374#discussion_r1322229618. @RussKie had local guidance for this repo in that comment thread.

Reply:
Oh thanks!